module/importer/src/Application/Controller/Api/Source/SourceCreateAction.php

```diff
@@ -98,22 +108,27 @@ public function __construct(
     public function __invoke(Request $request): Response
     {
         $type = $request->get('type');
+        $typeForm = $this->formFactory->create(SourceTypeForm::class);
```

Comment:
Isn't this a bit of an overkill to create artificial form to jut check one value? Can't this just be verified here? Just wondering.